tests/suites/test_suite_md.function

```diff
@@ -190,9 +190,10 @@ void md_text_multi( char *text_md_name, char *text_src_string,
     int halfway, len;
 
     const mbedtls_md_info_t *md_info = NULL;
-    mbedtls_md_context_t ctx;
+    mbedtls_md_context_t ctx, ctx_copy;
 
     mbedtls_md_init( &ctx );
+    mbedtls_md_init( &ctx_copy );
 
     memset(md_name, 0x00, 100);
     memset(src_str, 0x00, 1000);
```

Comment:
In the lines already there.. You are right of course, but I wasn't necessarily doing a run of fixing things in the code that were already present.. But I'll do it now anyway..